src/main/java/tsunami/security/scanner/utilities/App.java

```diff
@@ -17,105 +17,57 @@
-      if (f.isFile()) {
-        /* TODO: parse all config files under a certain app name to Kubernetes Object here*/
-        System.out.println(f);
-      }
-    }
-
-    /* Load mysql config file
-    * TODO: The method of getting file paths using user.dir will be resolved after previous TODO is done.
-    */
-    String mysqlConfigPath =
-        System.getProperty("user.dir") + "/application/wordpress/mysql-deployment.yaml";
-    File mysqlConfig = new File(mysqlConfigPath);
-    ImmutableList<Object> objectList4Mysql =
-        ImmutableList.copyOf((List<Object>) Yaml.loadAll(mysqlConfig));
-
-    /* Load wordpress config file */
-    String wordpressConfigPath =
-        System.getProperty("user.dir") + "/application/wordpress/wordpress-deployment.yaml";
-    File wordpressConfig = new File(wordpressConfigPath);
-    ImmutableList<Object> objectList4Wordpress =
-        ImmutableList.copyOf((List<Object>) Yaml.loadAll(wordpressConfig));
-
-    // TODO: Replace passwords in deployment configs.
 
+    // Initialize Kubernetes Java Client Api.
     ApiClient client = Config.defaultClient();
     Configuration.setDefaultApiClient(client);
 
-    // TODO: Using a map from Object.getClass() to different api calls. Still IN PROCESS.
-
-    /* Initialize apis */
-    CoreV1Api coreV1Api = new CoreV1Api();
-    AppsV1Api appsV1Api = new AppsV1Api();
-
-    /* Parse mysql configs */
-    V1Service mysqlSvc = (V1Service) objectList4Mysql.get(0);
-    V1PersistentVolumeClaim mysqlPvc = (V1PersistentVolumeClaim) objectList4Mysql.get(1);
-    V1Deployment mysqlDeployment = (V1Deployment) objectList4Mysql.get(2);
-
-    /* Parse wordpress configs */
-    V1Service wordpressSVC = (V1Service) objectList4Wordpress.get(0);
-    V1PersistentVolumeClaim wordpressPVC = (V1PersistentVolumeClaim) objectList4Wordpress.get(1);
-    V1Deployment wordpressDeployment = (V1Deployment) objectList4Wordpress.get(2);
-
-    /* Create services */
-    V1Service createResultSvc =
-        coreV1Api.createNamespacedService("default", mysqlSvc, null, null, null);
-    V1Service createResultWordpressSvc =
-        coreV1Api.createNamespacedService("default", wordpressSVC, null, null, null);
-
-    /* Create persistent volume claims */
-    V1PersistentVolumeClaim createResultPvc =
-        coreV1Api.createNamespacedPersistentVolumeClaim("default", mysqlPvc, null, null, null);
-    V1PersistentVolumeClaim createResultWordpressPvc =
-        coreV1Api.createNamespacedPersistentVolumeClaim("default", wordpressPVC, null, null, null);
-
-    /* Deploy */
-    V1Deployment createResultDeployment =
-        appsV1Api.createNamespacedDeployment("default", mysqlDeployment, null, null, null);
-    V1Deployment createResultWordpressDeployment =
-        appsV1Api.createNamespacedDeployment("default", wordpressDeployment, null, null, null);
+    // Initialize Util tool for Java Client Api.
+    KubeJavaClientUtil kubeUtil = new KubeJavaClientUtil();
```

Comment:
Is this line necessary? It seems that all methods in KubeJavaClientUtil are static.